server_apps/data_transfer/SWISS-PROT/sp_load.sql

```diff
@@ -121,7 +121,7 @@ alter table tmp_uniprot_db_link_with_dups
 
 --!echo 'Attribute db links to the internal pub record'
 	insert into record_attribution (recattrib_data_zdb_id, recattrib_source_zdb_id)
-		select dblink_zdb_id, 'ZDB-PUB-020723-2'
+		select dblink_zdb_id, 'ZDB-PUB-230615-71'
```

Comment:
I assume this change is on purpose. A new publication.

Reply:
Yes, Ken created a new publication to associate with UniProt accessions going forward.